cpp/tests/utilities/column_utilities.cu

```diff
@@ -58,7 +73,7 @@ public:
 };
 
 void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs, bool print_all_differences) {
-  expect_column_properties_equal(lhs, rhs);
+  expect_column_properties_equal_semantic(lhs, rhs);
```

Comment:
Shouldn't there be the option to expect null equality too?

Reply:
Possibly.  I think Jake was of the opinion nullability was irrelevant in all cases where you just care about actual values, not whether the columns are nullable.  The example being: column A and B are absolutely identical, except column A has validity data (all true) and B does not.  For the purposes of "expect_columns_equal" there's no difference.  Adding  @jrhemstad

Reply:
I just don't see the point of adding the bool parameter to `expect_column_properties_equal` if it's never used.

FWIW in most of my tests I ensure the expected column has the expected null state...